lib/Zureg/Model.hs

```diff
@@ -65,9 +77,8 @@ data RegisterInfo = RegisterInfo
     , riTShirt        :: !(Maybe (TShirtCut, TShirtSize))
     , riMentor        :: !Bool
     , riProject       :: !Project
-    -- | This field is optional because we only added after registration had
-    -- been opened.
-    , riRegisteredAt  :: !(Maybe Time.UTCTime)
+    , riRegisteredAt  :: !Time.UTCTime
+    , riHackathon      :: !Hackathon
```

Comment:
This PR looks good to me, one note I have is that having the `Hackathon` in the `RegisterInfo` doesn't seem necessary, since you would typically create a new database for every Hackathon?

Reply:
You're right, I changed it so that the `Hackathon` is read from configuration instead.